python/tskit/trees.py

```diff
@@ -3457,15 +3457,17 @@ def edge_diffs(self):
         descending parent time, parent id, then child_id). This means that within
         each list, edges with the same parent appear consecutively.
 
-        This iterator terminates after the final interval in the tree sequence
-        (i.e. it does not report a final removal of all remaining edges); the
-        number of iterations is thus always equal to the number of trees in the
-        tree sequence.
-
+        :param bool include_terminal: If False (default), the iterator terminates
+            after the final interval in the tree sequence (i.e. it does not
+            report a final removal of all remaining edges), so the number
+            of iterations will be equal to the number of trees in the tree
+            sequence. If True, there is one more iteration, with the last
+            ``edges_out`` value reporting all the edges contained in the final
```

Comment:
What's be value of left and right here?

Reply:
Good point. By default it is `right==left==sequence_length`, which seems reasonable to me, so I've documented this and added to to the high_level tests.